File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/request/inventory/ConsumeItemBody.java
package com.xsolla.android.store.entity.request.inventory;

import com.google.gson.annotations.SerializedName;

public class ConsumeItemBody {

    private String sku;
    private long quantity;
    @SerializedName("instance_id")
    private String instanceId;

    public ConsumeItemBody(String sku, long quantity, String instanceId) {
        this.sku = sku;
        this.quantity = quantity;
        this.instanceId = instanceId;
    }
}

File: xsolla-store-sdk/src/main/java/com/xsolla/android/store/entity/request/inventory/ConsumeItemBody.kt
package com.xsolla.android.store.entity.request.inventory

import com.google.gson.annotations.SerializedName

data class ConsumeItemBody(
    val sku: String,
    val quantity: Int? = null,
    @SerializedName("instance_id")
    val instanceId: String? = null
)